pkg/apis/pulumi/shared/stack_types.go

```diff
@@ -5,6 +5,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Adding the two new API fields for our Stack CR: `Targets` and `Prerequisite`. This is how our users will use this added feature.